Fedoraware/TeamFortress2/TeamFortress2/Features/Misc/Misc.cpp

```diff
@@ -510,86 +531,96 @@ void CMisc::BypassPure()
 		static DWORD dwAddress = 0x0;
```

Comment:
way better way to do this btw

Reply:
i didn't change this, only refactored

Reply:
i know im just saying https://www.unknowncheats.me/forum/team-fortress-2-a/487251-disabling-sv_pure.html this way is cooler and you should do this instead of this (this function is ran in paint btw for whatever reason????)

Reply:
i'll have a look at it later

Reply:
go use super cool new hooks :DDD